build.sbt

```diff
@@ -10,12 +10,10 @@ import scala.xml.transform.{RewriteRule, RuleTransformer}
 organization in ThisBuild := "org.http4s"
 version      in ThisBuild := scalazCrossBuild("0.15.2-SNAPSHOT", scalazVersion.value)
 apiVersion   in ThisBuild := version.map(extractApiVersion).value
-scalaVersion in ThisBuild := "2.11.8"
 // The build supports both scalaz `7.1.x` and `7.2.x`. Simply run
 // `set scalazVersion in ThisBuild := "7.2.4"` to change which version of scalaz
 // is used to build the project.
 scalazVersion in ThisBuild := "7.1.11"
-crossScalaVersions in ThisBuild := Seq("2.10.6", scalaVersion.value, "2.12.1")
```

Comment:
No, the sbt-travisci plugin reads this setting out of .travis.yml so it's not duplicated.  It's pretty neat.